helpers/dbload.ts

```diff
@@ -7,26 +7,22 @@ import { ChallengeTypes, Challenge } from './models/Challenge'
 import { AlertTypes, Alert } from './models/Alert'
 import { StarTypes, Star } from './models/Star'
 
-const options: Options = {
-  host: process.env.DB_HOST || 'city',
-  logging: false,
-  dialect: 'postgres',
-  pool: {
-    max: 5,
-    min: 0,
-    acquire: 30000,
-    idle: 10000
-  }
-}
-
-if (process.env.NODE_ENV != 'production') {
-  options.port = ((process.env.DB_PORT as unknown) as number) || 5432
-}
 const sequelize = new Sequelize(
-  process.env.DB_NAME || 'you',
-  process.env.DB_USER || 'failed',
-  process.env.DB_PW || 'this',
-  options
+  process.env.DB_NAME as string,
```

Comment:
fields of the process.env object are already strings, so the typecasting shouldnt be needed

Reply:
It's `string|undefined` (in case if env variable wasn't configured) so this type assertion is needed.

Reply:
In that case it'd be better to be explicitly clear and give it a default string value in case it is undefined. 
Like 
```
process.env.DB_NAME || 'you'
```
This change will also allow us to use only javascript, which simplifies things.

Reply:
Oh hmm I see that in our config file, we already set default values for all the env variables, so it would actually make contextual sense to use `as string` in these instances

Reply:
Is it possible for us to create typescript definitions for our env variables such that they are `string` instead of `string | undefined` ? I think that would increase the clarity of our codebase